dune-project

```diff
@@ -35,6 +35,6 @@
   result
   (ocaml-version
    (>= 2.3.0))
-  (odoc (>= 1.4.1))
+  (odoc-parser (>= 2.0.0))
```

Comment:
This makes Mdx incompatible with Odoc 1.5. Odoc 2.0.0 should be fully retrocompatible. The only reason I can think of a user keeping the current version is the change in the output.

Reply:
This is fixed now. Odoc 1.5 is co-installable with odoc-parser.